dev.ares.ares.yaml

```diff
@@ -2,6 +2,8 @@ app-id: dev.ares.ares
 runtime: org.freedesktop.Platform
 runtime-version: '23.08'
 sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
```

Comment:
Is Rust nightly a hard requirement, or could the library be built with Rust stable?

Reply:
Awesome, thanks for checking with them